package/CHANGELOG

```diff
@@ -80,7 +80,8 @@ Enhancements
   * Added dihedrals.py with Dihedral, Ramachandran, and Janin classes to
     analysis module (PR #1997, PR #2033)
   * Added the analysis.data module for reference data used in analysis (PR #2033)
-  * Added dihedrals.py with Ramachandran class to analysis module
```

Comment:
@zemanj I think you accidentally killed this entry

Reply:
I thought that's already included three lines above... am I wrong?

Reply:
ah it's weird it's in the log twice then, yeah ok